_topic_map.yml

```diff
@@ -366,14 +366,18 @@ Topics:
 - Name: Ingress
   Dir: ingress
   Topics:
-  - Name: Ingress Operator in OpenShift Container Platform
-    File: configuring-ingress-operator
-  - Name: Configuring the default certificate
-    File: configuring-default-certificate
-  - Name: Ingress controller configuration
-    File: ingress-controller-configuration
```

Comment:
Above three topics should be in `File: ingress-operator`.

Reply:
I made that change in the previous revision to this PR: the `ingress-operator` assembly includes the `nw-ingress-setting-a-custom-default-certificate` as well as the `nw-scaling-ingress-controller` module that was in the `ingress-controller-configuration` assembly.  Is that what you mean, or should I make some further changes?